### ISSUE TITLE
Bug Report: Game still stutters even after 0.5.2 has released

### ISSUE THREAD

**Issue by FunkinCentral** [FunkinCrew/Funkin#3667]
### Issue Checklist

- [X] I have properly named my issue
- [X] I have checked the Issues/Discussions pages to see if my issue has already been reported

### Platform

Itch.io (Downloadable Build) - Windows

### Browser

None

### Version

0.5.2

### Description (include any images, videos, errors, or crash logs)

While 0.5.1 had unplayable stutters, 0.5.2 has the same problem. There are still stutters randomly throughout the songs, (my input offset is always set to 0ms so it's not that.) the stutters are not as major as the last update, but they are still annoying and make it difficult to hit notes.

### Steps to Reproduce

1. Play any song

**Comment by trayfellow** [comment]
I can confirm this still happens to me too, i had a recording of Ugh Erect stuttering but github's 10mb filesize limit prevented me from reporting this aswell

I had my FPS set to 120 and Visual offsets and Input offsets set to default (0)

**Comment by Johnsen290** [comment]
just as I posted the same thing lol but yeah, even changing the visual offsets to 0 doesn't change anything, had a hundred FPS too

**Comment by FunkinCentral** [comment]
I'm so glad other people are having this issue I thought I was aboutta get bullied for having a bad computer 😭

**Comment by Johnsen290** [comment]
regardless, the audio shouldn't stutter at all whatever's the case, just the visuals

**Comment by MAZ12211** [comment]
I can confirm this as well but you should post a video showcasing it

**Comment by FunkinCentral** [comment]
cant post a video, github limit wont let me

**Comment by Johnsen290** [comment]
> cant post a video, github limit wont let me

compress it then, freeconvert has memory settings

**Comment by biomseed** [comment]
> cant post a video, github limit wont let me

Use [this](8mb.video)

**Comment by MAZ12211** [comment]
I'd record it, but it's best if the devs see it in front of them instead of scrolling all the way down lol

**Comment by oddMLan** [comment]
Can confirm this is still an issue on 0.5.2, commit e66290c according to the build watermark.
It's really bad with a 60fps cap. At 120fps it's less noticeable, but still present occasionally.

**Comment by JVNpixels** [comment]
Wasn’t able to reproduce, even though my PC is low-end, but my friend has a high-end pc and was having this issue.

**Comment by amyspark-ng** [comment]
it's because this hasn't been fixed in 0.5.2 because they haven't found a solution, please close as this is a replicate of https://github.com/FunkinCrew/Funkin/issues/3495, the issue is closed but the issue persists

**Comment by biomseed** [comment]
> it's because this hasn't been fixed in 0.5.2 because they haven't found a solution, please close as this is a replicate of #3495, the issue is closed but the issue persists

This shouldn't be closed if #3495 is closed.

**Comment by amyspark-ng** [comment]
it will probably be opened again

**Comment by AbnormalPoof** [comment]
> This shouldn't be closed if https://github.com/FunkinCrew/Funkin/issues/3495 is closed.

Issues can be reopened.

**Comment by MAZ12211** [comment]
I think it has something to do with FPS
I set mine to the max and it stutters less than with 60 FPS

**Comment by SEBBY180** [comment]
> I think it has something to do with FPS I set mine to the max and it stutters less than with 60 FPS

Yeah, the higher the FPS is set to, the less it happens.

**Comment by MAZ12211** [comment]
I hope they fix it soon 🙏

**Comment by amyspark-ng** [comment]
> > I think it has something to do with FPS I set mine to the max and it stutters less than with 60 FPS
> 
> Yeah, the higher the FPS is set to, the less it happens.

this is true! i have 60hz and have the game runnning a 300fps at it doesn't stutter anymore

**Comment by Crystalwarrior** [comment]
can confirm the stuttering is really bad on every song. Makes the game infuriating to play.
I also notice the stuttering only happens on every major beat. I'm gonna hazard a wild guess and the stuttering might have something to do with loading the next set of notes for some reason?

**Comment by AbnormalPoof** [comment]
Can't reproduce on macOS, but is this (yet again) another resync issue?

**Comment by MAZ12211** [comment]
Yeah
It's a frustrating issue

**Comment by CoolName3** [comment]
this happened to me too when 0.5.2 released I tried it with 60 fps and it was stuttering too much but then the next day it was barely stuttering and my fps was still 60

**Comment by DarkWeBareBears69** [comment]
That happened to me too when I tired to play the mod with some re-modified things.
https://x.com/CasperBunny2001/status/1845044709804405190

**Comment by Beelijah** [comment]
this is the only reason i play on v0.5.0 instead of the latest version (v0.5.3)

**Comment by AbnormalPoof** [comment]
Closing this as a duplicate of #3495. However, #3955 might've fixed this issue, try it out.
